package.json

```diff
@@ -116,5 +116,23 @@
   },
   "lint-staged": {
     "*.js": "eslint"
+  },
+  "release": {
+    "plugins": [
+      "@semantic-release/commit-analyzer",
+      "@semantic-release/release-notes-generator",
+      "@semantic-release/npm",
+      "@semantic-release/github"
+    ],
+    "branches": [
+      "v6",
+      {
+        "name": "v7",
+        "prerelease": "alpha"
+      }
+    ]
+  },
+  "publishConfig": {
+    "tag": "alpha"
```

Comment:
I guess when we merge this to the new v6 branch, we'll have to change this to minor as a one-off thing

Reply:
latest that is

Reply:
Ah, this is specific for the v7 branch?

Reply:
I see you've done this already so thanks!